lib/libutee/include/pta_rtc.h

```diff
@@ -0,0 +1,103 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2022, Microchip
+ */
+#ifndef __PTA_RTC_H
+#define __PTA_RTC_H
+
+#include <tee_api_types.h>
+
+#define PTA_RTC_UUID { 0xf389f8c8, 0x845f, 0x496c, \
+		{ 0x8b, 0xbe, 0xd6, 0x4b, 0xd2, 0x4c, 0x92, 0xfd } }
+
+#define PTA_RTC_INFO_VERSION		0x1
+
+/*
+ * RTC provides set/get offset and thus command PTA_CMD_RTC_GET_OFFSET and
+ * PTA_CMD_RTC_SET_OFFSET might be called
+ */
+#define PTA_RTC_FEATURE_CORRECTION	BIT(0)
+
+struct pta_rtc_time {
+	uint32_t tm_sec;
+	uint32_t tm_min;
+	uint32_t tm_hour;
+	uint32_t tm_mday;
+	uint32_t tm_mon;
+	uint32_t tm_year;
+	uint32_t tm_wday;
+} __packed;
```

Comment:
is `__packed` needed? I remember @jenswi-linaro worrying about such useless attribute.

Reply:
In theory no since both OP-TEE and the OS runs on the same platform. I added them because I was asked to on some other PR which add the same kind of communication pattern.

Reply:
Will wait for @jenswi-linaro answer before removing them.